src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    display: block;
+    background: url(../images/star.svg) center center no-repeat;
+    width: 16px;
+    height: 16px;
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
```

Comment:
What about other modifiers? It should work correct with different amount of stars